src/context/ConstantContext.tsx

```diff
@@ -74,7 +105,7 @@ export const ConstantStateProvider = (
          * userId & configureSession should be used together to create a group channel on the client side.
          */
         userId: props.userId,
-        configureSession: props.configureSession,
+        configureSession: props.configureSession ? configureSession : undefined,
```

Comment:
hmm why can't it be just `configureSession: configureSession`?

Reply:
Cause we're using trying to use the custom one if props.configureSession is provided?

Reply:
if props.configureSession not given,  new SessionHandler() will be set to configureSession. So I thought  we can just inject configureSession to provider.

Reply:
configureSession is optional. If it’s not provided, it shouldn't be included.